fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java

```diff
@@ -5069,4 +5088,53 @@ public int getFollowerCount() {
         }
         return count;
     }
+
+    public AnalysisJobScheduler getAnalysisJobScheduler() {
+        return analysisJobScheduler;
+    }
+
+    // TODO:
+    //  1. handle partition level analysis statement properly
+    //  2. support sample job
+    //  3. support period job
+    public void createAnalysisJob(AnalyzeStmt analyzeStmt) {
```

Comment:
How about move this function to StatisticsJobManager or AnalysisJobScheduler?

Reply:
Previous one will be removed in the future, and move to latter would break the single-responsibility principle

Reply:
Why

Reply:
how to create analysis job is not the responsibility of Env, Env should just work like a proxy